frontend/src/views/camp/SideBarPeriods.vue

```diff
@@ -16,7 +16,9 @@ Displays periods of a single camp.
           two-line>
           <v-list-item-content>
             <v-list-item-title>{{ item.description }}</v-list-item-title>
-            <v-list-item-subtitle>{{ new Date(item.start) }} - {{ new Date(item.end) }}</v-list-item-subtitle>
+            <v-list-item-subtitle>
+              {{ $moment.utc(period.start).format($tc('global.moment.dateShort')) }} - {{ $moment.utc(period.end).format($tc('global.moment.dateShort')) }}
```

Comment:
Should be `item.start` and `item.end`, right?

Reply:
Cood catch :sweat_smile: